src/test/scala/chiselTests/FixedPointSpec.scala

```diff
@@ -0,0 +1,46 @@
+// See LICENSE for license details.
+
+package chiselTests
+
+import chisel3._
+import chisel3.testers.BasicTester
+import org.scalatest._
+
+//scalastyle:off magic.number
+
```

Comment:
Done, though appearing where it does it is for entire file, not somehow linked to the class.  Magic numbers should not be considered anyway in test code, but IntelliJ doesn't seem to see the scalastyle-test-config.xml which does not carry this constraint, and since I rely on IntelliJ to do my style analysis, it is worth including for me